server/datasources.json

```diff
@@ -1,6 +1,6 @@
 {
   "db": {
-    "connector": "memory",
-    "defaultForType": "db"
+    "name": "db",
```

Comment:
FWIW, this looks like a bug of loopback-workspace to me. The `name` property should not be included in the JSON output, as the name is already stored as the key (top-level property name).

No need to fix it here though.